main_test.go

```diff
@@ -98,3 +100,14 @@ func TestMissingKeys(t *testing.T) {
 
 	assert.Equal(t, 20, realMain(echoChan))
 }
+
+func TestPluginManager(t *testing.T) {
```

Comment:
is this different from a plugin manager test?

Reply:
This is WIP. I'm thinking of how to mock an external plugin. Well, I'm just looking for a project where this was done before :)